src/60_Samples_%26_Templates/Housing.html

```diff
@@ -253,32 +284,33 @@ <h4>Mortgage Calculator</h4>
       </amp-accordion>
     </div>
 
-    <h4 class="m1">Properties sold nearby</h4>
-
     <!-- ## Properties sold nearby -->
     <!-- Use `amp-list` to show a list of properties sold nearby. Learn more about `amp-list` [here](/components/amp-list/) -->
-    <amp-list width="auto"
-              height="150"
-              layout="fixed-height"
-              src="/json/other_properties.json">
-      <template type="amp-mustache">
-        <a href="#" class="ampstart-card related m1">
-          <amp-img width="122"
-                   height="100"
-                   src="{{src}}"
-                   alt="{{alt}}"
-                   attribution="{{attribution}}"></amp-img>
-                 <p>{{desc}}</p>
-                 <p>{{address}}</p>
-                 <p class= "price-other">{{price}}</p>
-        </a>
-      </template>
-      <div overflow role="button" aria-label="Show more" class="list-overflow">
-        Show more
-      </div>
-    </amp-list>
+    <div>
+      <h4>Properties sold nearby</h4>
```

Comment:
change h4 to h2 for semantic consistency.

Reply:
done